Sources/PreferencesSwiftUI/PreferencePane+SwiftUI.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  PreferencePane+SwiftUI.swift
+//  Preferences
+//
+//  Created by Kacper on 06/01/2020.
+//
+
+import Foundation
+import SwiftUI
+
+/**
+	SwiftUI equivelent of PreferencePane protocol
+*/
+@available (macOS 10.15, *)
+public protocol PreferencePaneView: View {
+	var preferencePaneIdentifier: PreferencePane.Identifier { get }
+	var preferencePaneTitle: String { get }
+	var toolbarItemIcon: NSImage { get }
```

Comment:
```suggestion
	var toolbarItemIcon: Image { get }
```

Reply:
But there's a problem, I don't see possibility to convert Image back to NSImage. And NSImage is required in `ToolbarItemStyleViewController` and for coexistance with AppKit's NSViewControllers

Reply:
That's true. Oh, never mind.